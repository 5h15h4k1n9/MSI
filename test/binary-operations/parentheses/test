(2 + 3) * 4
10 / (2 + 3)
(5 > 3) && (2 < 1)
exit
