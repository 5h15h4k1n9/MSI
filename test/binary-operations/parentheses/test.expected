(int) 20
(int) 2
(bool) false
Bye-bye!
